utils/misc.mli

```diff
@@ -607,6 +607,11 @@ val get_build_path_prefix_map: unit -> Build_path_prefix_map.map option
 (** Returns the map encoded in the [BUILD_PATH_PREFIX_MAP] environment
     variable. *)
 
+val invert_build_path_prefix_map: string -> string list
+(** Returns the potential paths (in decreasing order of priority) from
```

Comment:
I would use "in priority order" as in build_path_prefix_map.mli, the notion of increasing/decreasing priority tends to be confusing and subject to opposite conventions.